src/qdisp/QueryRequest.cc

```diff
@@ -329,7 +329,7 @@ bool QueryRequest::_importStream(JobQuery::Ptr const& jq) {
     int len = expectedLen;
     const char* buff = GetMetadata(len);
     if (len != expectedLen) {
-        throw Bug("_importStream wrong header size=" + to_string(len) + " expected=" + to_string(expectedLen));
+        throw Bug("_importStream metadata wrong header size=" + to_string(len) + " expected=" + to_string(expectedLen));
```

Comment:
I would instead recommend using an `Issue` as a base class for all exceptions and `ERR_LOC` macro to avoid any explicit names or `string(__func__)`: https://github.com/lsst/qserv/blob/main/src/util/Issue.h

Reply:
Yes, this would be a better option.